docs/UserGuide.md

```diff
@@ -27,7 +27,7 @@ It is optimized for use via a **Command Line Interface** (CLI) while still havin
 
 If you can type fast, IH can get your Interview management tasks done faster than traditional GUI apps.
 
-Let's get started by following the [Quick Start](#Quick Start) section!
+Let's get started by following the [Quick Start](#QuickStart) section!
```

Comment:
For the links to work it has to be in all lowercase and spaces denoted as '-' symbols

So in this case it would be #quick-start

Reply:
Noted, will make the necessary changes